app/models/support/gds/user_facing_components.rb

```diff
@@ -1,5 +1,3 @@
-require 'support/gds/user_facing_component'
```

Comment:
We've got future commits that remove the redundant requires, so for consistency we should move this change (and the others in this commit that aren't just about moving the files) into those commits.